### PR TITLE
Proof of concept for running unit tests through helix

### LINKED ISSUE #3140: Move tests to helix
See dotnet/winforms-designer#1732 for more details about the proof of concept that has been done and the remaining work

### PR DESCRIPTION
**Pull request by AdamYoblick**
Fixes #3140 
Fixes https://github.com/dotnet/winforms-designer/issues/1732

This work is tracked by https://github.com/dotnet/winforms-designer/issues/1732 in the winforms-designer repo.

This task is mostly done, but there are still some remaining tasks:

1. Scaling to other OS's if we choose to (this is easy, just add to the `_HelixTargetQueues` variables in the yaml.
1. Fixing the few unit tests that are failing
1. Fixing the integration tests.
   * This one is the biggest task, as it's not as simple as unit tests. See https://github.com/dotnet/winforms-designer/issues/1732#issuecomment-729954799 for details.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/4259)

### CODE REVIEW

File: eng/SendToHelix.proj [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <HelixTargetQueues>Windows.10.Amd64.Open</HelixTargetQueues> <!-- Change to Windows.10.Amd64 for internal builds. See helix.dot.net for a list of queues -->
+    <Creator>winforms</Creator> <!-- Must provide for public builds, must be empty for internal builds -->
+    <HelixAccessToken></HelixAccessToken> <!-- Must provide for internal builds, must be empty for public builds -->
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>runtime</DotNetCliPackageType>
+    <DotNetCliVersion>$(MicrosoftNETCoreAppPackageVersion)</DotNetCliVersion>
+    <EnableXUnitReporter>true</EnableXUnitReporter>
+    <XUnitPublishTargetFramework>$(TargetFramework)</XUnitPublishTargetFramework>
+    <XUnitRuntimeTargetFramework>netcoreapp2.0</XUnitRuntimeTargetFramework>
+    <XUnitRunnerVersion>$(XUnitRunnerConsoleVersion)</XUnitRunnerVersion>
```
Comment (weltkante) [inline eng/SendToHelix.proj:45]: The old `helixpublish.proj` which this PR removes did use a variable for the runtime target framework, was it intentional that it is now pinned to a specific version while still keeping variables for the other properties?

Comment (RussKie) [reply eng/SendToHelix.proj:45]: `<XUnitRuntimeTargetFramework>$(XUnitRuntimeTargetFramework)</XUnitRuntimeTargetFramework>` this one?
Looking through the configs and binlogs, I don't believe `$(XUnitRuntimeTargetFramework)` was ever set... ¯\\_(ツ)\_/¯

Comment (weltkante) [reply eng/SendToHelix.proj:45]: Yeah thats what I meant, it looked off, but a quick search for the key turned up [this comment](https://github.com/dotnet/arcade/issues/4839#issuecomment-585967306) so it seems `netcoreapp2.0` is the "latest" version, even for .NET 5 as implied in [this discussion](https://github.com/dotnet/efcore/pull/22058)

Comment (RussKie) [reply eng/SendToHelix.proj:45]: Largely the current changes are working, I'm getting some random test failures, that aren't observable on non-Helix infra. Investigating.

**Pull request closed without merging**